wiki/Installation/en.md

```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
 5.  osu! will prompt you to sign in or register.
     -   If you have an account already, sign in.
     -   If you need to register, see [Registration](/wiki/Registration), then sign in.
+    
+### MacOS
```

Comment:
move this section to a `wiki/Installation/Mac_OS/en.md article 

https://gist.github.com/TPGPL/1beea5d9293dfe5e5781f8e29060d2e4 - article's content